be/src/connector/deletion_vector/deletion_bitmap.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "deletion_bitmap.h"
+
+#include "column/vectorized_fwd.h"
+
+namespace starrocks {
+
+StatusOr<bool> DeletionBitmap::fill_filter(uint64_t start, uint64_t end, Filter& filter) {
+    if (start >= end) {
+        return false;
+    }
+    roaring64_iterator_t* it = roaring64_iterator_create(_bitmap);
+    if (!roaring64_iterator_move_equalorlarger(it, start)) {
+        return false;
+    }
+
+    while (roaring64_iterator_has_value(it) && roaring64_iterator_value(it) < end) {
+        uint64_t value = roaring64_iterator_value(it);
+        filter[value - start] = 0;
+        roaring64_iterator_advance(it);
+    }
+
+    roaring64_iterator_free(it);
+    return true;
```

Comment:
If set no value to filter, return false or true?

Reply:
roaring64_iterator_move_equalorlarger would return false and return early